cli/cluster/cmd/replicaset.go

```diff
@@ -73,30 +73,27 @@ type PromoteCtx struct {
 }
 
 // pickPatchKey prompts to select a key to patch the config.
-func pickPatchKey(keys []string, force bool, pathMsg string) (int, error) {
-	if len(keys) == 0 {
+// If force is true, picks the first passed key.
+func pickPatchKey(opts replicaset.PickPatchKeyOpts) (int, error) {
```

Comment:
Or I missed something?
```suggestion
func pickPatchKey(keys []string, force bool) (int, error) {
```

Reply:
Yes, It's a typo. Corrected.